### PR TITLE
Add arguments to synopsis sections of CLI ref

### LINKED ISSUE #17398: Add arguments in synopsis sections of CLI ref pages
https://github.com/dotnet/docs/pull/15694#discussion_r366601946

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0ba6c9b5-64cb-1dc6-21d0-ad8f0a1185fe
* Version Independent ID: 0f6c658c-b477-2d2c-2a9a-d74f6ae38d3f
* Content: [dotnet command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet#feedback)
* Content Source: [docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #17398 

Also put the options section of dotnet-install-scripts in alphabetical order.

### CODE REVIEW

File: docs/core/tools/dotnet-install-script.md [hunk 1]
```diff
@@ -59,34 +78,41 @@ You can install a specific version using the `-Version|--version` argument. The
...
+  Disables downloading from the [Azure Content Delivery Network (CDN)](https://docs.microsoft.com/azure/cdn/cdn-overview) and uses the uncached feed directly.
 
-  Installs just the shared runtime bits, not the entire SDK. This option is equivalent to specifying `-Runtime|--runtime dotnet`.
+- **`-NoPath|--no-path`**
+
+  If set, the installation folder isn't exported to the path for the current session. By default, the script modifies the PATH, which makes the .NET Core CLI available immediately after install.
+
+- **`-ProxyAddress`**
+
+  If set, the installer uses the proxy when making web requests. (Only valid for Windows)
```
Comment (gewarren) [inline docs/core/tools/dotnet-install-script.md:111]:
```suggestion
  If set, the installer uses the proxy when making web requests. (Only valid for Windows.)
```

File: docs/core/tools/dotnet-install-script.md [hunk 2]
```diff
@@ -59,34 +78,41 @@ You can install a specific version using the `-Version|--version` argument. The
...
+
+- **`ProxyUseDefaultCredentials`**
+
+  If set, the installer uses the credentials of the current user when using proxy address. (Only valid for Windows)
```
Comment (gewarren) [inline docs/core/tools/dotnet-install-script.md:115]:
```suggestion
  If set, the installer uses the credentials of the current user when using proxy address. (Only valid for Windows.)
```

File: docs/core/tools/dotnet-install-script.md [hunk 3]
```diff
@@ -96,53 +122,38 @@ You can install a specific version using the `-Version|--version` argument. The
   - `aspnetcore` - the `Microsoft.AspNetCore.App` shared runtime.
   - `windowsdesktop` - the `Microsoft.WindowsDesktop.App` shared runtime.
 
-- **`-DryRun|--dry-run`**
-
-  If set, the script won't perform the installation. Instead, it displays what command line to use to consistently install the currently requested version of the .NET Core CLI. For example, if you specify version `latest`, it displays a link with the specific version so that this command can be used deterministically in a build script. It also displays the binary's location if you prefer to install or download it yourself.
+- **`--runtime-id <RID>`**
 
-- **`-NoPath|--no-path`**
+  Specifies the [runtime identifier](../rid-catalog.md) for which the tools are being installed. Use `linux-x64` for portable Linux. (Only valid for Linux/macOS)
```
Comment (gewarren) [inline docs/core/tools/dotnet-install-script.md:127]:
```suggestion
  Specifies the [runtime identifier](../rid-catalog.md) for which the tools are being installed. Use `linux-x64` for portable Linux. (Only valid for Linux/macOS.)
```

File: docs/core/tools/dotnet-new.md [hunk 4]
```diff
@@ -115,15 +119,15 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
...
   Specifies a NuGet source to use during install. Available since .NET Core 2.1 SDK.
 
 - **`-o|--output <OUTPUT_DIRECTORY>`**
 
   Location to place the generated output. The default is the current directory.
 
-- **`--type`**
+- **`--type <TYPE>`**
 
   Filters templates based on available types. Predefined values are "project", "item", or "other".
```
Comment (gewarren) [inline docs/core/tools/dotnet-new.md:132]: Should the values be code fenced?

File: docs/core/tools/dotnet-run.md [hunk 5]
```diff
@@ -14,10 +14,13 @@ ms.date: 02/19/2020
 ## Synopsis
 
 ```dotnetcli
-dotnet run [-c|--configuration] [-f|--framework] [--force] [--interactive] [--launch-profile]
-    [--no-build] [--no-dependencies] [--no-launch-profile] [--no-restore] [-p|--project]
-    [-r|--runtime] [-v|--verbosity] [[--] [application arguments]]
-dotnet run [-h|--help]
+dotnet run [-c|--configuration <CONFIGURATION>] [-f|--framework <FRAMEWORK>]
+[--force] [--interactive] [--launch-profile <NAME>] [--no-build]
```
Comment (gewarren) [inline docs/core/tools/dotnet-run.md:18]: Indent the options?

File: docs/core/tools/dotnet-tool-update.md [hunk 6]
```diff
@@ -15,18 +15,18 @@ ms.date: 02/14/2020
...
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
 dotnet tool update <PACKAGE_NAME>
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+    [--configfile <FILE>] [--framework <FRAMEWORK>]
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
-dotnet tool update <-h|--help>
+dotnet tool update -h|--help
```
Comment (gewarren) [inline docs/core/tools/dotnet-tool-update.md:29]: The angle brackets you removed appear on line 18 and 21 in this file, and also in dotnet-tool-uninstall.md. Should they be removed there too?

**Review by gewarren — APPROVED** [review]
LG, left some suggestions.

File: docs/core/tools/dotnet-new.md (hunk 4 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-new.md:132]: Yes, good point.

Comment (tdykstra) [inline docs/core/tools/dotnet-new.md:132]:
```suggestion
  Filters templates based on available types. Predefined values are `project`, `item`, or `other`.
```

File: docs/core/tools/dotnet-run.md (hunk 5 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-run.md:18]: Yes.
```suggestion
[--force] [--interactive] [--launch-profile <NAME>] [--no-build]
```

File: docs/core/tools/dotnet-run.md [hunk 7]
```diff
@@ -14,10 +14,13 @@ ms.date: 02/19/2020
...
+[--no-dependencies] [--no-launch-profile] [--no-restore]
+[-p|--project <PATH>] [-r|--runtime <RUNTIME_IDENTIFIER>]
+[-v|--verbosity <LEVEL>] [[--] [application arguments]]
```
Comment (tdykstra) [inline docs/core/tools/dotnet-run.md:21]:
```suggestion
    [--no-dependencies] [--no-launch-profile] [--no-restore]
    [-p|--project <PATH>] [-r|--runtime <RUNTIME_IDENTIFIER>]
    [-v|--verbosity <LEVEL>] [[--] [application arguments]]
```

File: docs/core/tools/dotnet-tool-update.md (hunk 6 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-tool-update.md:29]: Yes, good catch.

File: docs/core/tools/dotnet-tool-update.md [hunk 8]
```diff
@@ -15,18 +15,18 @@ ms.date: 02/14/2020
 
 ```dotnetcli
 dotnet tool update <PACKAGE_NAME> <-g|--global>
```
Comment (tdykstra) [inline docs/core/tools/dotnet-tool-update.md:17]:
```suggestion
dotnet tool update <PACKAGE_NAME> -g|--global
```

File: docs/core/tools/dotnet-tool-update.md [hunk 9]
```diff
@@ -15,18 +15,18 @@ ms.date: 02/14/2020
...
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+    [--configfile <FILE>] [--framework <FRAMEWORK>]
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
-dotnet tool update <PACKAGE_NAME> <--tool-path>
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+dotnet tool update <PACKAGE_NAME> <--tool-path <PATH>>
```
Comment (tdykstra) [inline docs/core/tools/dotnet-tool-update.md:21]:
```suggestion
dotnet tool update <PACKAGE_NAME> --tool-path <PATH>
```

File: docs/core/tools/dotnet-tool-uninstall.md [hunk 10]
```diff
@@ -16,11 +16,11 @@ ms.date: 02/14/2020
 ```dotnetcli
 dotnet tool uninstall <PACKAGE_NAME> <-g|--global>
```
Comment (tdykstra) [inline docs/core/tools/dotnet-tool-uninstall.md:17]:
```suggestion
dotnet tool uninstall <PACKAGE_NAME> -g|--global
```

File: docs/core/tools/dotnet-tool-uninstall.md [hunk 11]
```diff
@@ -16,11 +16,11 @@ ms.date: 02/14/2020
...
 
-dotnet tool uninstall <PACKAGE_NAME> <--tool-path>
+dotnet tool uninstall <PACKAGE_NAME> <--tool-path <PATH>>
```
Comment (tdykstra) [inline docs/core/tools/dotnet-tool-uninstall.md:19]:
```suggestion
dotnet tool uninstall <PACKAGE_NAME> --tool-path <PATH>
```

**Comment by tdykstra** [comment]
@gewarren Thanks for the review!

**Pull request merged**